common/app/views/mainAMP.scala.html

```diff
@@ -40,6 +42,22 @@
 
             @body
 
+            <div class="fc-container__inner">
+                <div class="fc-container__header__title">
+                    <span>popular</span>
+                </div>
+                <amp-list width="600" height="1000" layout="responsive" src="https://www.thegulocal.com//most-read-amp.json">
```

Comment:
Thanks @gidsg. updated, but unsure of whether it is the correct page still, as I just make it relative.